src/components/ProductCard/ProductCard.css

```diff
@@ -15,18 +15,23 @@
 }
 
 .card-img {
-  aspect-ratio: 1; 
-  object-fit: contain; 
+  aspect-ratio: 1;
+  object-fit: contain;
 }
 .card-content {
   color: var(--text-color500);
 }
+
+.card-content:hover {
+  backdrop-filter: none;
+}
+
 .card-description {
   height: 7rem;
 }
 
 .card:hover .card-content {
-  background: rgba(255,255,255,0.5);
+  background: rgba(255, 255, 255, 0.7);
```

Comment:
if possible use color variable

Reply:
This css gives a glassy look to my content which is not possible with hex values.